docs/ops/detection/ExperimentalDetectronGenerateProposalsSingleImage_6.md

```diff
@@ -0,0 +1,107 @@
+## ExperimentalDetectronGenerateProposalsSingleImage <a name="ExperimentalDetectronGenerateProposalsSingleImage"></a> {#openvino_docs_ops_detection_ExperimentalDetectronGenerateProposalsSingleImage_6}
+
+**Versioned name**: *ExperimentalDetectronGenerateProposalsSingleImage-6*
+
+**Category**: Object detection
+
+**Short description**: An operation *ExperimentalDetectronGenerateProposalsSingleImage* computes ROIs and their scores based on input data.
+
+**Detailed description**: Operation doing next steps:
+
+1.  Transposes and reshape predicted deltas and scores to get them into the same order as the anchors;
+2.  Transforms anchors into proposals and clips proposals to image;
+3.  Removes predicted boxes with either height or width < *min_size*;
+4.  Sorts all `(proposal, score)` pairs by score from highest to lowest;
+5.  Takes top *pre_nms_count* proposals;
+6.  Applies non-maximum suppression with *nms_threshold*;
+7.  Takes *post_nms_count* proposals and return these top proposals and their scores.
+       
+**Attributes**:
+
+* *min_size*
+
+    * **Description**: *min_size* attribute specifies minimum box width & height.
+    * **Range of values**: non-negative floating point number
+    * **Type**: float
+    * **Default value**: 0.0
+    * **Required**: *yes*
+
+* *nms_threshold*
+
+    * **Description**: *nms_threshold* attribute specifies threshold to be used in the NMS stage.
+    * **Range of values**: non-negative floating point number
+    * **Type**: float
+    * **Default value**: 0.7
+    * **Required**: *yes*
+
+* *post_nms_count*
+
+    * **Description**: *post_nms_count* attribute specifies number of top-n proposals after NMS.
+    * **Range of values**: non-negative integer number
+    * **Type**: int
+    * **Default value**: 1000
+    * **Required**: *yes*
+
+* *pre_nms_count*
+
+    * **Description**: *pre_nms_count* attribute specifies number of top-n proposals before NMS.
+    * **Range of values**: non-negative integer number
+    * **Type**: int
+    * **Default value**: 1000
+    * **Required**: *yes*
```

Comment:
If this attribute is required, the we needn't specify the default value.

Reply:
Sure, deleted.